aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -101,7 +128,7 @@ SparseTensor& log1p_out_sparse(const SparseTensor& t, SparseTensor& r) {
 }
 
 SparseTensor log1p_sparse(const SparseTensor& t) {
-  auto result = get_result_tensor_for_unary_op(t);
+  auto result = get_result_tensor_for_unary_op<true>(t);
```

Comment:
When the default values are removed use comments before the template values for readability:

```
/*promotes_integer_to_float*/=true, /*is_linear_map*/=false
```